tools/modules/EnsEMBL/Web/Object/VEP.pm

```diff
@@ -502,7 +502,7 @@ sub species_list {
         'variation'   => $db_config->{'DATABASE_VARIATION'} // undef,
         'refseq'      => $db_config->{'DATABASE_OTHERFEATURES'} && $sd->get_config($_, 'VEP_REFSEQ') // undef,
         'assembly'    => $sd->get_config($_, 'ASSEMBLY_NAME') // undef,
-        'regulatory'  => $sd->get_config($_, 'REGULATORY_BUILD') // undef,
+        'regulatory'  => $db_config->{'DATABASE_FUNCGEN'}{'tables'}{'regulatory_build'}{'analyses'}{'Regulatory_Build'}->{'count'} // undef,
```

Comment:
Why the need to change this?

Reply:
Hi @nuno-agostinho , this is for automated detection of species that have regulatory build. Currently we rely on INI file but with this change we do not have to update ini file manually, web VEP will automatically pick up species that we can run `--regulatory` on.

I have removed these changes for now - as after discussion with Garth, they would review the config and make update if needed (basically we would not rely on `analyses` table in future). We can automate this part when we get confirmation. The [PR](https://github.com/Ensembl/public-plugins/pull/766) is targeted for e113 now.